runtime/src/main/codestarts/quarkus/quakus-hilla-maven-codestart/codestart.yml

```
@@ -0,0 +1,20 @@
+name: quarkus-hilla-maven-codestart
+ref: quarkus-hilla-maven-starter
+type: code
+tags:
+  - extension-codestart
+  - maven-only
+
+metadata:
+  title: Quarkus-Hilla starter
```

```suggestion
  title: Quarkus-Hilla Starter
```
I would name it all uppercase, looks and reads better (personal opinion)

---

done